kite:
    # Publish all joint states -----------------------------------
    joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50

    # Position Controllers ---------------------------------------
    gimbal_joint1_position_controller:
      type: effort_controllers/JointPositionController
      joint: gimbal_joint1
      pid: {p: 5000.0, i: 100.0, d: 120.0}
      
    gimbal_joint2_position_controller:
      type: effort_controllers/JointPositionController
      joint: gimbal_joint2
      pid: {p: 5000.0, i: 100.0, d: 120.0}
    
    briddle_x_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: briddle_x_joint
      pid: {p: 300.0, i: 10.0, d: 10.0}
      
    briddle_y_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: briddle_y_joint
      pid: {p: 300.0, i: 1.0, d: 10.0}
      
    briddle_z_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: briddle_z_joint
      pid: {p: 300.0, i: 1.0, d: 10.0}
      
    tether_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: tether_joint
      pid: {p: 4000.0, i: 1.0, d: 10.0}
